libdevcore/Common.h

```diff
@@ -135,6 +135,16 @@ inline u256 s2u(s256 _u)
 		return u256(c_end + _u);
 }
 
+inline std::ostream& operator<<(std::ostream& os, bytes const& _bytes)
```

Comment:
I find the current implementation most useful in my fixed point work. But i could definitely see where toHex would also be useful. Hrmmm....I can compromise here.